lib/internaltypes.d.ts

```diff
@@ -0,0 +1,89 @@
+import { Temporal } from '..';
+export type BuiltinCalendarId =
```

Comment:
This is now the source of truth for what the registered calendars are, right? It looks like the `impl` within calendar.ts will now (correctly) reject assigning any key to it that isn't listed here?

Reply:
Yes.